### DISCUSSION TITLE
Femur dataset on ShapeWorks Portal

### DISCUSSION THREAD

**Discussion by JodriOost** [SCIInstitute/ShapeWorks#950] [Q&A] [answered]
Hi

I just want to know what the femur dataset for the use-case-data dataset on the ShapeWorks Portal consists of? Does the dataset consist of healthy femurs or are there pathological femurs included in the dataset? Sorry for the inconvenience, but the dataset does not include a License.txt file, which describes the terms of use and citation requirements for the data.

**Comment by jadie1** [comment, marked as answer]
Hi, the dataset consists of all normal/healthy femurs. We have added a License.txt file with the terms of use to the portal.
